lib/backend/ndb/rpmxdb.c

```diff
@@ -228,16 +265,10 @@ static int rpmxdbReadHeader(rpmxdb xdb)
     if (!slotnpages || !pagesize || stb.st_size % pagesize != 0)
 	return RPMRC_FAIL;
     xdb->pagesize = pagesize;
+    xdb->mapflags = xdb->rdonly ? PROT_READ : PROT_READ | PROT_WRITE, MAP_SHARED;
```

Comment:
This generates a warning:
```
backend/ndb/rpmxdb.c: In function ‘rpmxdbReadHeader’:
backend/ndb/rpmxdb.c:268:69: warning: right-hand operand of comma expression has no effect [-Wunused-value]
  268 |     xdb->mapflags = xdb->rdonly ? PROT_READ : PROT_READ | PROT_WRITE, MAP_SHARED;
      |    
```

I assume the ',' is supposed to be a '|' instead :)